snips_nlu/cli/inference.py

```diff
@@ -33,10 +35,15 @@ def parse(training_path, query, verbose=False):
     elif verbose >= 2:
         set_nlu_logger(logging.DEBUG)
 
+    if intents_filter:
+        intents_filter = intents_filter.split(",")
```

Comment:
Since it's not forbidden to have `,` in the intent name, you should provide some way to escape this special character. Maybe the standard lib `csv` module will help you do that easily

Reply:
indeed